README.md

```diff
@@ -42,6 +42,7 @@ Special thanks to the [ACE3 Team](http://ace3mod.com/team.html) for their open s
 - Vehicle customization garage made specifically for Zeus.
 - Various bug fixes and quality of life improvements to the Zeus interface.
 - Settings to control Zeus camera properties such as speed and available vision modes.
+- Zeus camera flashlight for easier nighttime editing.
```

Comment:
```suggestion
- Zeus camera flashlight for easier night-time editing.
```

Reply:
Both are correct, but "... for easier editing at night." might be best?

Reply:
I'd prefer the one with the hyphen because it's easier to read, but your variant would work too.

Reply:
I'd go with mharis' one, sounds a bit nicer.

Reply:
```suggestion
- Zeus camera flashlight for easier editing at night.
```